### DISCUSSION TITLE
Incorrect information in stats - My total PR's and commit's number is not updating

### DISCUSSION THREAD

**Discussion by Deva-1903** [anuraghazra/github-readme-stats#2352] [Q&A] [answered]
### Describe the bug

![Screenshot (2)](https://user-images.githubusercontent.com/97371915/209757094-3efc4d34-3655-4160-9af5-961da5484016.png)
**Incorrect information in stats - My total PR's and commit's number is not updating**

**Comment by rickstaa** [comment, marked as answer]
@Deva-1903 Thanks for your question. I checked your results in the [GraphQL backend](https://docs.github.com/en/graphql/overview/explorer), but the results on the card look correct:

```graphql
query userInfo($login: String!) {
  user(login: $login) {
    name
    login
    contributionsCollection {
      totalCommitContributions
      restrictedContributionsCount
    }
  }
}
```

```
{"login": "Deva-1903"}
```

It shows 328 public commits and 19 private commits. The only thing that causes your results not to show up immediately is that we use a 4-hour caching period (see https://github.com/anuraghazra/github-readme-stats#common-options). Further, you must set the ` &count_private=true` variable and [need to share private commits on your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile) for them to show up on the card (see https://github.com/anuraghazra/github-readme-stats/discussions/1770#how-to-count-private-stats).
